tensorflow/contrib/rnn/python/ops/core_rnn_cell_impl.py

```diff
@@ -83,12 +83,14 @@ def call(self, inputs, state):
 class GRUCell(RNNCell):
   """Gated Recurrent Unit cell (cf. http://arxiv.org/abs/1406.1078)."""
 
-  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None):
+  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None,
+               initializer=None):
     super(GRUCell, self).__init__(_reuse=reuse)
     if input_size is not None:
       logging.warn("%s: The input_size parameter is deprecated.", self)
     self._num_units = num_units
     self._activation = activation
+    self._initializer = initializer
```

Comment:
Please add 2 initializer arguments: kernel_initializer and bias_initializer (which will need to be passed to linear). That'll be compatible with current tf.layers. (We might want more kinds of initializers layer, but at least as a start this looks good.)